doc/changes/changes_1.0.2.md

```diff
@@ -4,6 +4,8 @@ Code name: Dependency Updates
 
 ## Summary
 
+**Aborted:** This release was aborted since the release process was broken. Use 1.0.3. instead.
```

Comment:
Should the release have made it half-way to GitHub, please don't forget to mark it as "deprecated".